packages/fluentui/react-northstar/package.json

```diff
@@ -52,7 +52,8 @@
     "simulant": "^0.2.2"
   },
   "files": [
-    "dist"
+    "dist",
+    "./README.md"
```

Comment:
You don't need this; the readme is included automatically regardless of `files` setting